a2t-whisper/docker/cpu/Dockerfile

```diff
@@ -1,31 +1,17 @@
 FROM python:3.10-slim-bullseye
-
-ARG MODEL_ID
-
 WORKDIR /usr/src/app/
 
-ENV PYTHONDONTWRITEBYTECODE 1
-ENV PYTHONUNBUFFERED 1
```

Comment:
was this actually required? seen it in a few places from the root commit on this repo @filopedraz